jf_agent/data_manifests/git/generator.py

```diff
@@ -46,20 +44,14 @@ def create_manifests(
         try:
             # If the git config doesn't have a git instance, do not generate a manifest
             if not instance_slug:
-                agent_logging.log_and_print(
-                    logger,
-                    logging.WARNING,
-                    f'Git instance for company {company_slug} was detected as NONE. The manifest for this instance will not be processed or uploaded. This should NOT affect your agent upload',
+                logger.debug(
```

Comment:
Q: Any reason why this is debug? This feels informational.

Reply:
Also this was previously "WARNING" level.

Reply:
Manifests for agent are still in a flaky alpha state, I've gotten some feed back from clients (via our client facing teams) that they get skittish about warnings and error messages, so hiding warnings for 'non essential code' like Manifests is the best design choice, IMO. We want to know if something goes wrong with manifests, but clients don't really care, and they might get worried if they see warning messages